demo/demo_process.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/python3
+
+'''
+This demo provides a processing script to calculate the energy
+for each phase based on the record time stamp.
+'''
+import numpy as np
+import os
+import time
+from ina219_pi_seelab import ina219_pi_seelab
+
+PWR_FILE = "./ina219_power.txt"
+LOG_FILE = "./ina219_log.txt"
+MEASURE_TIME = 10
+SAMPLE_INTERVAL = 100 # ms
+MSG = "Collecting power measurements for {} seconds...\r\n".format(
+        MEASURE_TIME)
+MSG += "Check {} for detailed traces afterwards.".format(PWR_FILE)
+
+def process():
+    '''
+    Calculate total and average power based on
+    power file and time stamp log file
+    '''
+    all_pwr = np.loadtxt(PWR_FILE, delimiter=',')
+    phase = np.loadtxt(LOG_FILE, delimiter=',')
+    i, start_idx, end_idx = 0, 0, 0
+    for start, end in phase:
+        phase_pwr = []
+        while i < len(all_pwr):
+            if all_pwr[i][0] == start:
```

Comment:
In reality, the exact time of `start` may not be a sample in `all_pwr`, so replace `==` with `>=` would be safer.